how-to-docs/kubernetes-token.md

```diff
@@ -1,15 +1,23 @@
 # How to get a Kubernetes token for the UK cluster
 
-Log into the [platform hub][platform hub link].
+## Users
 
-Select **Support Requests** under **Help & Support** from the sidebar.
+1. Talk to your project admin and ask them to generate a user token for you.
```

Comment:
How do I know who is an admin for the project? Maybe give them some steps to get to the relevant project page and see who has the "Admin" tag.